subsys/logging/backends/log_backend_uart.c

```diff
@@ -103,27 +118,36 @@ static int char_out(uint8_t *data, size_t length, void *ctx)
 	return length;
 }
 
-static uint8_t uart_output_buf[CONFIG_LOG_BACKEND_UART_BUFFER_SIZE];
-LOG_OUTPUT_DEFINE(log_output_uart, char_out, uart_output_buf, sizeof(uart_output_buf));
-
 static void process(const struct log_backend *const backend,
 		union log_msg_generic *msg)
 {
+	const struct lbu_cb_ctx *ctx = backend->cb->ctx;
```

Comment:
at some point the `backend->cb->ctx` becomes NULL, no sure why it happens yet

Reply:
Found it:

![image](https://github.com/zephyrproject-rtos/zephyr/assets/1563974/132b821d-fb71-4c90-adf2-bfbda51120d0)